lib/python_helper_sympy.py

```diff
@@ -86,7 +130,9 @@ def __init__(self):
 #     https://github.com/newville/asteval
 
 class Error(Exception):
-    def __init__(self, offset):
+    def __init__(self, offset, extra_msg = None):
+        if extra_msg != None:
+            self.message += " [EXTRA-MSG: %s]" % extra_msg
```

Comment:
@jonatanschroeder 
Right. I believe at the time I was trying to debug how to make a symbolic function with no body for the implementation behave as expected using the sympy constructs. At one point, IIRC, something was getting printed, but it just clutters things up now. It will be removed the next time I make a commit to this branch for the PR.